### DISCUSSION TITLE
Welcome to cei-web-portal Discussions!

### DISCUSSION THREAD

**Discussion by MGeovany** [MGeovany/cei-web-portal#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wpayze** [comment]
Aquí vemos claramente una imagen en donde si la analizamos con detenimiento y a profundidad, podemos observar que a partir de los distintos puntos de discrepancia y comparación se puede llegar a la idea central que se aleja del argumento principal, de lo cual concluimos que este comentario no tiene sentido alguno. Pero quizás no sea importante para muchos de ustedes y se pierda entre los demás comentarios, la mayoría ni lo leerá, tal vez sea criticado y con el tiempo sea olvidado, pero en fin, sólo quería decirles que vendo empanadas

**Reply by MGeovany** [reply]
grax
